lua/telescope/config.lua

```diff
@@ -428,9 +428,18 @@ append(
       - hook(s):          Function(s) that takes `(filepath, bufnr, opts)`
                           to be run if the buffer previewer was not shown due to
                           the respective test.
-                          Available hooks are: {mime, filesize, timeout}_hook, e.g.
+                          Available hooks are: {mime, filesize, timeout}_hook;
+                          for instance, for too text large files you can get the first
```

Comment:
"for too text large files" -> "for too large text files"

Reply:
haha, thanks :)